app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
+
+
+def calculate_team_total_rating(team: List) -> int:
+    total_rating = 0
+    for player in team:
+        total_rating += player.get_rating()
+    return total_rating
```

Comment:
Implement this method in one line using `sum` as [mentioned in the checklist](https://github.com/Volokitin-Vitaliy/py-elves-and-dwarves/blob/master/checklist.md#code-efficiency)